MinecraftClient/Protocol/ProtocolHandler.cs

```diff
@@ -704,7 +704,7 @@ private static int DoHTTPSGet(string host, string endpoint, string cookies, ref
             http_request.Add("User-Agent: Java/1.6.0_27");
             http_request.Add("Accept-Charset: ISO-8859-1,UTF-8;q=0.7,*;q=0.7");
             http_request.Add("Connection: close");
-            http_request.Add("");
+            http_request.Add("\r\n");

```

Comment:
```suggestion
            http_request.Add("");
            http_request.Add("");
```
Headers are joined using `\r\n` so adding an additional blank line will add an additional `\r\n`:
```C#
String.Join("\r\n", headers.ToArray()
```